api/src/main/java/org/pmiops/workbench/institution/InstitutionEmailDomainMapper.java

```diff
@@ -1,14 +1,22 @@
 package org.pmiops.workbench.institution;
 
+import static org.pmiops.workbench.institution.InstitutionUtils.getEmailDomainsByTierOrEmptySet;
```

Comment:
opt nit: personally I'm not a big fan of static imports outside of test utilities, but we don't have a firm rule around this in the codebase.